.github/workflows/integ-tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Run Integration Tests
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  integ-tests:
+    runs-on: ubuntu-latest
+    environment:
+      name: integ-tests
+    steps:
+        - uses: actions/checkout@v3
+        - uses: actions/setup-python@v5
+          with:
+            python-version: '3.11'
+        - name: allows us to build amr64 images
```

Comment:
Dose this mean we need to test RIE in both x86 and arm?

Reply:
Yes